torchvision/models/detection/_utils.py

```diff
@@ -468,3 +468,31 @@ def retrieve_out_channels(model: nn.Module, size: Tuple[int, int]) -> List[int]:
         model.train()
 
     return out_channels
+
+
+@torch.jit.unused
+def _onnx_tracing_topk_min(input: Tensor, orig_kval: int, axis: int) -> Tuple[int, int]:
+    """
+    ONNX spec requires the k-value to be less than or equal to the number of inputs along
+    provided dim. Certain models uses python's min() function to determine whether to use
+    the provided k-value or the specified dim axis value.
```

Comment:
> "Certain models uses python's min() function to determine whether to use the provided k-value or the specified dim axis value."?

This statement is not very clear. I think you are trying to say that certain models use the number of elements along a particular `axis` instead of `K` if `K` exceeds the number of elements along that axis?

Reply:
Yup, you are right. Will change the wording to reflect this.